app/api/server/v1/invites.js

```diff
@@ -0,0 +1,57 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../api';
+import { findOrCreateInvite } from '../../../invites/server/functions/findOrCreateInvite';
+import { listInvites } from '../../../invites/server/functions/listInvites';
+import { useInviteToken } from '../../../invites/server/functions/useInviteToken';
+import { validateInviteToken } from '../../../invites/server/functions/validateInviteToken';
+
+API.v1.addRoute('listInvites', { authRequired: true }, {
```

Comment:
Will this list all the invites for all the rooms for a normal user?

Reply:
@pierre-lehnen-rc what about this review?

Reply:
It'll list every invite for users who can create invites